readme.md

```diff
@@ -291,9 +293,28 @@ If you are new to eBPF, you may want to try the links described as "introduction
   - [The adventures of a Suricate in eBPF land](http://netdevconf.org/1.2/slides/oct6/10_suricata_ebpf.pdf)
   - [eBPF and XDP seen from the eyes of a meerkat](https://www.slideshare.net/ennael/kernel-recipes-2017-ebpf-and-xdp-eric-leblond)
+- [Project Calico](https://projectcalico.docs.tigera.io/about/about-calico) - Calico is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. Calico's eBPF data plane delivers a low latency, high throughput data plane with a rich network security policy model.
+  - [Enabling eBPF data plane with Calico](https://projectcalico.docs.tigera.io/maintenance/ebpf/enabling-bpf)
+- [merbridge](https://github.com/merbridge/merbridge/) - Use eBPF to speed up your Service Mesh. Merbridge replaces iptables rules with eBPF to intercept traffic. It also combines msg_redirect to reduce latency with a shortened datapath between sidecars and services.
+
+### Observability
 
 - [InKeV: In-Kernel Distributed Network Virtualization for DCN](https://github.com/iovisor/bpf-docs/blob/master/university/sigcomm-ccr-InKev-2016.pdf)
+- [DEEP-mon](https://www.slideshare.net/necstlab/deepmon-dynamic-and-energy-efficient-power-monitoring-for-containerbased-infrastructures) - Helps with measuring power consumption for servers and uses eBPF programs for in-kernel aggregation of data.
+- [pixie](https://github.com/pixie-io/pixie) - Observability for Kubernetes using eBPF. Features include protocol tracing, application profiling, and support for distributed bpftrace deployments.
+- [SkyWalking Rover](https://github.com/apache/skywalking-rover) - [Apache SkyWalking](https://skywalking.apache.org/) is an open-source Application Performance Monitoring (APM) platform specially designed for distributed systems with microservices, cloud-native and container-based (Kubernetes) architectures. SkyWalking Rover is an eBPF-based profiler and metrics collector for C, C++, Golang, and Rust applications.
+- [parca-agent](https://github.com/parca-dev/parca-agent) - eBPF based always-on continuous profiler for analysis of CPU and memory usage, down to the line number and throughout time.
+- [rbperf](https://github.com/javierhonduco/rbperf) - Sampling profiler and tracer for Ruby.
+- [hubble](https://github.com/cilium/hubble) - Network, Service & Security Observability for Kubernetes using eBPF
+- [caretta](https://github.com/groundcover-com/caretta) - Instant K8s service dependency map generated by eBPF, right to your Grafana.
+
+### Library
+
 - [gobpf - utilizing eBPF from Go](https://fosdem.org/2017/schedule/event/go_bpf/) - A library to create, load and use eBPF programs from Go.
+- [eBPF for Windows](https://github.com/microsoft/ebpf-for-windows) - This project is a work-in-progress that allows using existing eBPF toolchains and APIs familiar in the Linux ecosystem to be used on top of Windows.
+
+### Tool
```

Comment:
All projects from ply down to greggd are about tracing, and fall under “Observability”. The rest can probably be split between networking (upf-bpf), tools and utilities (redbpf). FUSE is the only one I'm not sure where to move, but then I'm also unsure what's the status on that project. Maybe an “Other” or “Miscellaneous” section? :thinking:

Reply:
I have some different ideas about the tracing tools. I consider ply as something like gdb. Normally people won't add gdb as part of their observability stack. So maybe it would be better to leave them here?

Reply:
As for FUSE with eBPF, this project is stopped since 2020: https://github.com/extfuse/extfuse.
So maybe we can remove it now?

Reply:
> Normally people won't add gdb as part of their observability stack.

OK that's maybe tracing/monitoring vs. observability. What if we call the section “Tracing & Observability”? For me, it all comes down to inspecting the kernel's internals, as opposed to networking or security enforcement for example. What do you think?

> As for FUSE

The project seems inactive, but not necessarily defunct. There are other entries that are not exactly up-to-date, either. I'd say, let's keep extFUSE until we do some larger spring cleaning.

Reply:
Looking again at the current sections, it looks good as well, let's keep the sections as they are. Maybe make `Tool` plural though (`Tools`).

Reply:
Regarding FUSE and BPF: https://patchwork.kernel.org/project/netdevbpf/list/?series=697889&state=*&archive=both